bin/simple-supply-lint

```diff
@@ -31,6 +31,7 @@ lint() {
 }
 
 ret_val=0
+export PYTHONPATH=$PYTHONPATH:$TOP_DIR/protobuf
```

Comment:
Does this mean you're running the linters on the generated protobuf files?

Reply:
No, this is just so that the generated protobufs are able to be imported while linting